dali/operators/audio/nonsilence_op.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/core/static_switch.h"
+#include "dali/operators/audio/nonsilence_op.h"
+#include "dali/pipeline/data/views.h"
+
+namespace dali {
+
+DALI_SCHEMA(NonsilenceRegion)
```

Comment:
```suggestion
DALI_SCHEMA(NonSilenceRegion)
```
I think would read better

Reply:
Actually, in english typically _non-_ with noun is written jointly, making this one word (e.g. _nonnegative_, _noncorformist_). That's why I went for `Nonsilence`.

Reply:
After consideration, I'll go with `NonsilentRegion`